src/components/QueryEditor/index.tsx

```diff
@@ -9,5 +9,9 @@ import { AdxDataSourceOptions, KustoQuery } from 'types';
 type Props = QueryEditorProps<AdxDataSource, KustoQuery, AdxDataSourceOptions>;
 
 export const QueryEditor: React.FC<Props> = (props) => {
-  return config.featureToggles.adxLegacyEditor ? <LegacyQueryEditor {...props} /> : <NewQueryEditor {...props} />;
+  return (config.featureToggles as any).adxLegacyEditor ? (
```

Comment:
Also, can you please leave a code comment to explain the use of `any` here? So that if anyone encounters this line in future, we can go back and check whether the issue preventing us from using `config.featureToggles` is solved.

Reply:
Added a comment